src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverterOfT.cs

```diff
@@ -139,7 +139,7 @@ internal virtual bool OnTryRead(ref Utf8JsonReader reader, Type typeToConvert, J
         /// <param name="reader">The <see cref="Utf8JsonReader"/> to read from.</param>
         /// <param name="typeToConvert">The <see cref="Type"/> being converted.</param>
         /// <param name="options">The <see cref="JsonSerializerOptions"/> being used.</param>
-        /// <returns>The value that was converted.</returns>
+        /// <returns>The value that was converted. <seealso cref="HandleNull"/> to determine if the converter handles a 'null' JSON value.</returns>
```

Comment:
A custom converter could return `null` regardless of whether it is handling null JSON tokens. I think the `HandleNull` comment should be moved to the <remarks/> section.

Reply:
Moving to remarks section. Thanks